resources/web/style/link.pcss

```diff
@@ -1,6 +1,6 @@
 #guide {
   a {
-    color: #0077CC;
+    color: #343741;
```

Comment:
I think we should use a different color for links outside of the table of contents. It's really difficult to determine if text is a link in the main content especially without `text-decoration`. I think I changed it from `#00a9e5` to `#0077CC` because it was bothering me that it was a slightly different shade of blue than than the color that's used in the buttons in the top nav. 😛 

For example there's no way to know all the values in the left column are links until you hover over them (I'm hovering over `cloudfront` in the screenshot below):

<img width="905" alt="image" src="https://user-images.githubusercontent.com/10479155/182202892-fac6724c-7178-4ae4-9fc1-26b2f10e872a.png">

Reply:
Oops! I didn't realize I changed these links too.